back/spec/acceptance/admin_publications_spec.rb

```diff
@@ -100,14 +100,52 @@
         json_response = json_parse(response_body)
 
         if CitizenLab.ee?
-          expect(json_response[:data].size).to eq 9
-          expect(json_response[:data].map { |d| d.dig(:relationships, :publication, :data, :id) }).to match_array [@empty_draft_folder.id, @folder.id, @projects[0].id, @projects[1].id, @projects[2].id, @projects[3].id, @projects[4].id, @projects[5].id, @projects[6].id]
+          expect(json_response[:data].size).to eq 8
```

Comment:
We want to return only the `admin_publications` representing folders _which contain projects for the specified areas_ (and top-level projects for the specified areas), when areas are specified  

 Thus, the empty folder this test expected is no longer expected to be returned in the response.